plugins/kagents/skills/pester-patterns/SKILL.md

```
@@ -171,3 +171,312 @@ Invoke-Pester -Configuration $config
+        $script:Module = Import-Module ./MyModule.psd1 -PassThru -Force
+    }
+
+    It 'Funktioniert' {
+        $script:Module.Version | Should -Not -BeNullOrEmpty
+    }
+}
+```
+
+### Ausführungsreihenfolge (Pester 5)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)